docs/concepts-guidelines.md

```diff
@@ -231,7 +231,7 @@ It will use their own fork to push code and create the pull request.
 
 1. Create a new GitHub user and login.
 2. Fork the repository that you will be creating pull requests in.
-3. Create a Classic [Personal Access Token (PAT)](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token) with `repo` scope.
+3. Create a Classic [Personal Access Token (PAT)](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token) with `repo` [and `workflow`](https://github.com/peter-evans/create-pull-request/issues/3510) scopes.
```

Comment:
Your edit would make it sound as if it was _always_ required, which is not the case. It's required if the upstream repository uses GitHub Actions, and therefore there will likely be changes to the `.github/workflows` path occasionally.

```suggestion
3. Create a Classic [Personal Access Token (PAT)](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token) with `repo` scope. If the upstream repository uses GitHub Actions, it's highly likely that you will also need the `workflow` scope.
```

By the way, this info also exists in a more general way in the main readme [here](https://github.com/peter-evans/create-pull-request#token), but I think it's good to be explicit in this section.

Reply:
I worded it intentionally like that. As noted in the initial comment:
> Even if the scenario I hit this with is fairly specific, we know that if nothing else, at least _this_ action is in .github/workflows somewhere, [...]

So it's all but certain that there *will* be some `.github/workflows` files because this `peter-evans/create-pull-request` is in use in the upstream repo. If it's not, there would be no need for this doc in the first place.

Reply:
Not necessarily. It's possible to combine `push-to-fork` with [creating pull requests in a remote repository](https://github.com/peter-evans/create-pull-request/blob/main/docs/concepts-guidelines.md#creating-pull-requests-in-a-remote-repository). So the remote and fork might not use GitHub Actions, even if the repository where the workflow runs does.

However, I should make that clear in the docs because I think that's the only case where the `workflow` scope won't be needed.

Reply:
You know what, just to make this less complicated I'm just going to say that you need the `workflow` scope, which will probably be the case for the majority of users. 

Updated here: https://github.com/peter-evans/create-pull-request/commit/25b6871a4ebe4c3585f47c7a687ac6fd0ec0e32d